src/Backend/Synapse/ToFinalTable/IncrementalImporter.php

```diff
@@ -69,14 +69,15 @@ public function importToTable(
             );
 
             $state->startTimer(self::TIMER_UPDATE_TARGET_TABLE);
-            $this->connection->executeStatement(
-                $this->sqlBuilder->getUpdateWithPkCommand(
-                    $stagingTableDefinition,
-                    $destinationTableDefinition,
-                    $options,
-                    $timestampValue
-                )
+            $sql = $this->sqlBuilder->getUpdateWithPkCommand(
+                $stagingTableDefinition,
+                $destinationTableDefinition,
+                $options,
+                $timestampValue
             );
+            if ($sql !== '') {
+                $this->connection->executeStatement($sql);
```

Comment:
nechces do toho ifu zabalit i ty timery? uplne nevim kde vybublaji, ale pokud nekde jo, tak tam bude zbytecnej a nulovej timer

Reply:
jo upravil sem to. Ono to prakticky nikdy nenastane, enom v inkrementálním loadu do WS a to sa prakticky nepoužívá